docs/lakebridge/docs/reconcile/recon_notebook.mdx

```diff
@@ -210,6 +210,73 @@ table_recon = TableRecon(
         )
     ]
 )
+
+```
+## Configure Recon Metadata and Dashboards using Notebook
+
+The recommended way is to use the CLI following the instructions [here](../installation#configure-reconcile).
+However, if you want to configure the metadata tables and dashboards using a notebook, you can use the following code snippet.
+This needs to be executed only once unless you want to change the metadata configuration.
+
+```python
+from importlib.resources import files
+from pathlib import Path
+
+import databricks.labs.lakebridge.resources
+from databricks.labs.lakebridge import __version__
+from databricks.labs.lakebridge.config import (
+    DatabaseConfig,
+    ReconcileConfig,
+    ReconcileMetadataConfig,
+)
+from databricks.labs.lakebridge.contexts.application import ApplicationContext
+from databricks.sdk import WorkspaceClient
+
+ws = WorkspaceClient(product="lakebridge", product_version=__version__)
+app_context = ApplicationContext(ws)
+resources = files(databricks.labs.lakebridge.resources)
+dashboards_dir = resources.joinpath("reconcile/dashboards")
+if not isinstance(dashboards_dir, Path):
+    raise TypeError("Dashboards directory is not a real filesystem path.")
+
+# Create a ReconcileConfig instance with the necessary configurations.
+# Make sure the objects exist.
+recon_config = ReconcileConfig(
+    data_source="oracle",
```

Comment:
Note: Can you use widgets instead, so they know they need to update the widgets
 
 Also look at Reconcile Automation page on how create a widget for this with import notebook link.

Reply:
@bishwajit-db  any update on this?